find-start-lms.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+RED="\x1B[31m"
+GREEN="\x1B[32m"
+NC="\x1B[0m"
+
+if [ -z ${DEVSTACK_WORKSPACE+x} ]; then DEVSTACK_WORKSPACE=$(pwd)/../../openedx; fi
```

Comment:
Doesn't the `+x` substitute `DEVSTACK_WORKSPACE` with `x` if it's not there? May want to be careful about changing `DEVSTACK_WORKSPACE` in general because devstack uses it a lot internally.

Reply:
i am aware, but it is pointing to the path correctly... as for `+x` i think shellcheck made me change it to this... ill look in to it again.

Reply:
`if [ -z ${var+x} ]; then echo "var is unset"; else echo "var is set to '$var'"; fi`

where ${var+x} is a [parameter expansion](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02) which evaluates to nothing if var is unset, and substitutes the string x otherwise.

From: https://stackoverflow.com/a/13864829

Do you have a method youd prefer to use for this @pshiu? Otherwise i think im ready to merge